iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.cs

```diff
@@ -137,20 +138,21 @@ private async Task<string> PrepareStorageForImportExportAsync(string storageAcco
         }
 
         /// <summary>
-        /// Create the SAS token for the container. 
+        /// Create the SAS token for the container.
         /// </summary>
-        /// <param name="container"></param>
-        /// <returns></returns>
         private string GetContainerSasToken(CloudBlobContainer container)
         {
             // Set the expiry time and permissions for the container.
             // In this case no start time is specified, so the
             // shared access signature becomes valid immediately.
-            var sasConstraints = new SharedAccessBlobPolicy();
-            sasConstraints.SharedAccessExpiryTime = DateTime.UtcNow.AddHours(24);
-            sasConstraints.Permissions = SharedAccessBlobPermissions.Write
-                | SharedAccessBlobPermissions.Read
-                | SharedAccessBlobPermissions.Delete;
+            var sasConstraints = new SharedAccessBlobPolicy
+            {
+                SharedAccessExpiryTime = DateTime.UtcNow.AddHours(24),
+                Permissions = SharedAccessBlobPermissions.Write
+                    | SharedAccessBlobPermissions.Read
+                    | SharedAccessBlobPermissions.Delete
+                    | SharedAccessBlobPermissions.Create,
```

Comment:
In my testing, I added create. I probably should remove it, as it doesn't seem to be necessary.